content/ja/docs/topics/advanced.md

```diff
@@ -50,23 +50,23 @@ bash では、これは `renderer1 | renderer2 | renderer3` のように単純
 `kustomize` をポストレンダラーとして使用する例は
 [here](https://github.com/thomastaylor312/advanced-helm-demos/tree/master/post-render)で見ることができます。
 
-### Caveats
-When using post renderers, there are several important things to keep in mind.
-The most important of these is that when using a post-renderer, all people
-modifying that release **MUST** use the same renderer in order to have
-repeatable builds. This feature is purposefully built to allow any user to
```

Comment:
> in order to have repeatable builds.

When I run your translation through google translate back to English, it looks like this concept might have been left out. This is the reason for everyone working on a release to use the same renderer. Can you confirm that this concept is included in your translation?